tide/core/PowerTimer.h

```diff
@@ -0,0 +1,74 @@
+#ifndef POWERTIMER_H
+#define POWERTIMER_H
+
+#include "serialization/includes.h"
+#include "types.h"
+
+#include <boost/enable_shared_from_this.hpp>
+
+#include <QObject>
+#include <QTimer>
+
+/**
+ * Inform user about inactivity timeout and allow to stop the power off process.
+ */
+class PowerTimer : public QObject,
+                   public boost::enable_shared_from_this<PowerTimer>
+{
+    Q_OBJECT
+    Q_DISABLE_COPY(PowerTimer)
+
+public:
+    /**
+     * Construct a timer used to power off Planar equipment.
```

Comment:
since this PowerTimer has been designed as separate from the PlanarController, replace "Planar equipment" by something generic like "displays" or "display hardware".